src/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { GetOwnedAssetsResult } from "./types";
+
+export function parseToGetOwnedAssetsResult(
+  result: unknown[]
+): GetOwnedAssetsResult {
+  return result as GetOwnedAssetsResult;
```

Comment:
this is a reminder for now to parse data returned by wallet to the expected format, even if not returned by wallet in the expected format.